components/src/core/components/Input/Select/select-input.scss

```diff
@@ -220,23 +219,52 @@
   border-radius: 0.85rem;
   &.button-double-line {
     padding: 0.5rem 0.75rem 0.5rem 1rem;
+    cursor: default !important;
```

Comment:
Can't we apply this change to SelectInputButton component itself rather than having to Select widget.

Reply:
Checked. this doesn't affects select input; this only applies the select input which has the new behaviour.

`&.button-double-line {`